src/azure/cli/commands/arm.py

```diff
@@ -90,6 +96,23 @@ def resource_exists(resource_group, name, namespace, type, **_): # pylint: disab
 
 def add_id_parameters(command_table):
 
+    def _extract_ids_help(command):
```

Comment:
@johanste @BurtBiel because the ID parameter is handled after aliasing has already occurred, any attempts to alias this parameter through `register_cli_argument` will be ignored. I use this approach to retrieve it after the fact (because the argument registry is still laying around). However, it slows the commands down noticeably even if you have not requested help (which is the only time you need it). Do you have any suggestions on how to (1) bring the ids parameter into the fold earlier so it can be aliased like normal or (2) reliably identify that the user has not requested help and skip terminate this method early.